com.unity.ml-agents/Runtime/Sensor/ISensor.cs

```diff
@@ -14,28 +24,28 @@ public interface ISensor
     {
         /// <summary>
         /// Returns the size of the observations that will be generated.
-        /// For example, a sensor that observes the velocity of a rigid body (in 3D) would return new {3}.
-        /// A sensor that returns an RGB image would return new [] {Width, Height, 3}
+        /// For example, a sensor that observes the velocity of a rigid body (in 3D) would return
+        /// new {3}. A sensor that returns an RGB image would return new [] {Width, Height, 3}
         /// </summary>
-        /// <returns></returns>
+        /// <returns>Size of the observations that will be generated.</returns>
         int[] GetObservationShape();
 
         /// <summary>
-        /// Write the observation data directly to the WriteAdapter.
-        /// This is considered an advanced interface; for a simpler approach, use SensorBase and override WriteFloats instead.
-        /// Note that this (and GetCompressedObservation) may be called multiple times per agent step, so should not
-        /// mutate any internal state.
+        /// Write the observation data directly to the <see cref="WriteAdapter"/>.
+        /// This is considered an advanced interface; for a simpler approach, use SensorBase
+        /// and override WriteFloats instead. Note that this (and GetCompressedObservation) may
```

Comment:
Can we cref SensorBase and WriteFloats? and maybe GetCompressedObservation?

Reply:
And actually, WriteFloats no longer exists; it's WriteObservation now. All the more reason to cref it.

Reply:
Done. Did a pass on the file wrt cref's.